library/templates/pharmacy_list_form.php

```diff
@@ -0,0 +1,415 @@
+                    <option value=""><?php echo text("State *"); ?></option>
+                    <?php while($row = sqlFetchArray($res)){ ?>
+                        <option value="<?php echo $row['option_id']; ?>"><?php echo xlt($row['title']); ?></option>
+                    <?php   
+                    }
+                    ?>
+                </select>
+            </div>
+
+            <div class="col px-0 mx-0">
+                <select class="form-control" name="weno_city" id="weno_city" onchange="cityChanged()"><?php echo text("Enter City *"); ?></select>
+                <div class="warn"></div>
+            </div>
+            <span class="ml-1">OR</span>
+            <div class="col">
+                <input class="form-control form-control-sm" name="weno_zipcode" id="weno_zipcode" placeholder="Zipcode">
+            </div>
+            <div>
+                <button type="button" class="btn btn-primary btn-sm" onclick="search()"><?php echo xlt("Search"); ?></button>
+            </div>
+        </div>
+
+        <div class="row col mt-2">
+            <div>
+                <select class="form-control form-control-sm" name="form_weno_pharmacy" id="weno_pharmacy" onchange="pharmSelChanged()">
+                    <option value=""></option>
+                </select>
+            </div>
+        </div>
+        
+    </form>
+    <div class="row mt-2 mb-1 contianer col-12">
+        <button type="button" class="btn btn-primary btn-sm mr-3" onclick="assignPrimaryPharmacy()"><?php echo text("Assign Primary Pharmacy"); ?></button>
+        <button type="button" class="btn btn-primary btn-sm" onclick="assignAlternatePharmacy()"><?php echo text("Assign Alternate Pharmacy"); ?></button>
+        <button type="button" class="btn btn-light btn-sm ml-3" onclick="resetForm()"><?php echo xlt("Reset"); ?></button>
+    </div>
+
+    <div class="row col-12 small mb-1">
+        <?php echo xlt("See below Weno Selected Pharmacies"); ?>
+    </div>
+    <div class="row col-12">
+        <span class="font-weight-bold"><?php echo xlt("Weno Selected Primary Pharmacy: "); ?></span>
+        <span id="weno_primary"></span>
+    </div>
+    <div class="row col-12">
+        <span class="font-weight-bold"><?php echo xlt("Weno Selected Alternate Pharmacy: "); ?></span>
+        <span id="weno_alt"></span>
+    </div>
+</body>
+</html>
+
+<script type="text/javascript">
+    var wenoState = null;
+    var wenoCity = null;
+    var wenoZipcode = null;
+    var coverage = null;
+    var fullDay = null;
+    var wenoOnly = null;
+    var testPharmacies = null;
+    var wenoPrimPharm = null;
+    var wenoValChanged= null;
+    var wenoAltPharm = null;
+    var requirdField = "Field is required";
+
+    window.onload = (event) => {
+        
+        let pid = <?php echo json_encode($pid); ?>;
+        let prevPrimPharmacy = <?php echo $prev_prim_pharmacy; ?>;
+        let prevAltPharmacy = <?php echo $prev_alt_pharmacy; ?>;
```

Comment:
run all 3 variables in 3 lines above through js_escape() (instead of json_encode)